plugins/Goals/vue/src/ManageGoals/ManageGoals.vue

```diff
@@ -41,8 +41,7 @@
                   :is="beforeGoalListActionsHeadComponent"
                 ></component>
 
-                <th v-if="userCanEditGoals">{{ translate('General_Edit') }}</th>
-                <th v-if="userCanEditGoals">{{ translate('General_Delete') }}</th>
+                <th>{{ translate('General_Actions') }}</th>
```

Comment:
Any reason for showing the column when the user doesn't have permission to perform any action?

Reply:
It's kind of the standard going forward and some tables have view actions. I can hide it if that's what you prefer.

Reply:
@Stan-vw @Javi-Ormaechea What do you think? Should the Actions column be hidden if the user doesn't have write access or should it always display?

Reply:
Makes sense to hide items when they are not relevant to the user, so indeed if the actions column has zero action options for a user we can hide the entire column 👍

Reply:
I made the change :+1:

Reply:
We can hide them or grey them out, latter one would be my preference as it would show users what's possible/doable on a specific table (creates awareness). Then they could either request to become an editor to perform these needed actions or ask the right person to go through them. Otherwise we can just hide them.

Reply:
Thanks @Javi-Ormaechea. As I'm guessing that we'll want to change the hover text to explain why the actions are disabled, I'm going to simply hide the actions for now.